charts/virtual-nodes/templates/node.yaml

```diff
@@ -0,0 +1,196 @@
+{{- else if eq $node.type "dgxh100.80g" }}
+  {{- $resources := deepCopy $defaultResources }}
+  {{- $resources = set $resources "cpu"                 224    }}
+  {{- $resources = set $resources "memory"              "2Ti"  }}
+  {{- $resources = set $resources "nvidia.com/gpu"      8      }}
+  {{- $resources = set $resources "nvidia.com/mlnxnics" 16     }}
+  {{- $resources = set $resources "ephemeral-storage"   "30Ti" }}
+  {{- $params = set $params "resources" $resources }}
+  {{- $params = set $params "nodeInstance" "dgxh100.80g.8.norm" }}
+  {{- $params = set $params "nodeGroup"    "gpu" }}
+  {{- $params = set $params "nodeType"     "gpu" }}
+
+# cpu.x86
+{{- else if eq $node.type "cpu.x86" }}
+  {{- $resources := deepCopy $defaultResources }}
+  {{- $resources = set $resources "cpu"               48             }}
+  {{- $resources = set $resources "memory"            "196692052Ki"  }}
+  {{- $resources = set $resources "ephemeral-storage" "2537570228Ki" }}
+  {{- $params = set $params "resources" $resources }}
+  {{- $params = set $params "nodeGroup" "cpu" }}
+  {{- $params = set $params "nodeType"  "cpu" }}
+
+{{- else }}
+  {{- $error := printf "Unsupported node type '%s'" $node.type }}
+  {{- fail $error }}
+{{- end }}
+
+{{- $count := ($node.count | int) }}
+{{- range until $count }}
+---
+apiVersion: v1
+kind: Node
+metadata:
+  annotations:
+    node.alpha.kubernetes.io/ttl: "0"
+    kwok.x-k8s.io/node: fake
+  labels:
+    beta.kubernetes.io/arch: amd64
+    beta.kubernetes.io/os: linux
+    kubernetes.io/arch: amd64
+    kubernetes.io/hostname: virtual-{{ $node.type }}-{{.}}
+    kubernetes.io/os: linux
+    kubernetes.io/role: agent
+    node-role.kubernetes.io/agent: ""
+    gpu_affinity_capable: "true"
+    node-role.bcp.ngc.nvidia.com/role: user-workload
+    nodeGroup: {{ $params.nodeGroup }}
+    nodeType: {{ $params.nodeType }}
+    {{- if $params.nodeInstance }}
+    nodeInstance: {{ $params.nodeInstance }}
+    {{- end }}
+    pod_gpu_size.static: dynamic
+    preempt_upon_cordon: "true"
+    type: kwok
+  name: virtual-{{ $node.type }}-{{.}}
+status:
+  conditions:
+  - type: AggregatedNodeHealth
+    reason: NodeReady
+    status: "True"
+    message: Node is healthy
+  - message: Flannel is running on this node
+    reason: FlannelIsUp
+    status: "False"
+    type: NetworkUnavailable
+  - message: acs kernel module is disabled
+    reason: ACSModuleDisabled
+    status: "False"
+    type: ACSModuleCheck
+  - message: kernel has no deadlock
+    reason: KernelHasNoDeadlock
+    status: "False"
+    type: KernelDeadlock
+  - message: Filesystem is read-only
+    reason: FilesystemIsReadOnly
+    status: "False"
+    type: ReadonlyFilesystem
+  - message: Flannel is running on this node
+    reason: FlannelDeviceAvailable
+    status: "False"
+    type: FlannelNetworkDeviceProblem
+  - message: Dgx has /raid
+    reason: DgxRaidOk
+    status: "False"
+    type: DgxRaidProblem
```

Comment:
We can remove this condition it was supposed for internal use only on a specific set of cluster nodes and not a standard condition.

Reply:
addressed